### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'delete_run'/'create_run' when running in colab

### ISSUE THREAD

**Issue by EvelynWang-0423** [jina-ai/finetuner#695]
When I tried demo notebook  <Finetuner C CLIP Benchmark>  in colab, trying to msater how to use Finetuner.
finetuner.delete_run('clip-run') & run = finetuner.fit
all ended up in error: 
 AttributeError: 'NoneType' object has no attribute 'delete_run'/'create_run'

But when running in local mechine, this error didn't exist. 

What can be done to fix this problem and enable me to run this notebook successfully on cloud. 
Thx in advance
![image](https://user-images.githubusercontent.com/84950843/225845371-b73047da-96a1-43df-8730-aa4926e5e96b.png)

**Comment by EvelynWang-0423** [comment]
FYI: Referring to the closed issues, I've checked that I've successfully logged in to JINA
![image](https://user-images.githubusercontent.com/84950843/225848101-3620c1cc-06d1-4167-be04-a0f5511c0102.png)

**Comment by guenthermi** [comment]
Can you try it with `finetuner.login(force=True)` instead of `finetuner.notebook_login()`

**Comment by EvelynWang-0423** [comment]
> Can you tell, which notebook you are using? You could try to run it with `finetuner.login(force=True)` instead of `finetuner.notebook_login()`. Maybe this solves the issue, however, the noteboks on the Readme already do this, so maybe it is a different error

Thx for your timely reply.

i've tried` finetuner.login(force=True) `instead of `finetuner.notebook_login()` already last time, and it returns the success of my log in. However, problems still exist.
![image](https://user-images.githubusercontent.com/84950843/226263000-dc9ae303-aa75-4b0b-a901-5186703ae894.png)

AttributeError: 'NoneType' object has no attribute 'create_run' still exists.

The notebook url: https://colab.research.google.com/drive/1fHSUML3UmrRltzn7xjgl5Db6QOqXjNHa#scrollTo=O7BbFKF71c-l

Thanks again for your help.

**Comment by guenthermi** [comment]
Hi, we noticed, that this notebook was using an old version of finetuner which is not compatible with the API anymore. We updated it. Now it should work again. However, since you copied from the old one, it is not working. Here is the new one: https://colab.research.google.com/github/jina-ai/workshops/blob/main/notebooks/finetuner-clip/FinetunerXCLIPBenchmark.ipynb?ref=jina-ai#scrollTo=FhXVm8tV0VEr
